### DISCUSSION TITLE
Code Flow Diagram of clm_drv

### DISCUSSION THREAD

**Discussion by adrifoster** [ESCOMP/CTSM#2140] [Show and tell]
In thinking about some refactoring with @johnpaulalex I created a code flow diagram for `clm_drv`.

https://miro.com/app/board/uXjVMqDTIiw=/?share_link_id=856640462812 

Please feel free to peruse!

**Comment by mvertens** [comment]
@adrifoster - thanks for doing this. However,  I'm not able to see anything when I click on the link.

**Reply by adrifoster** [reply]
Hmm, that's odd I am able to access it from an Incognito Chrome window and a Safari window... @billsacks says it's working for him.

**Reply by ekluzek** [reply]
It does work for me as well. It does take a bit for it to render though. Is there some firewall issue or something going to Norway?

**Reply by rosiealice** [reply]
Not for the whole of Norway at any rate ;) (I can see it from my house!)

**Reply by mvertens** [reply]
I'm able to see it now. No idea what happened this morning. Very nifty! Would love to know how you created it in more detail.
